android/app/src/main/java/com/expensify/chat/StartupTimer.java

```diff
@@ -0,0 +1,39 @@
+package com.expensify.chat;
+import android.util.Log;
+
+import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.bridge.ReactContextBaseJavaModule;
+import com.facebook.react.bridge.ReactMethod;
+import com.google.firebase.perf.FirebasePerformance;
+import com.google.firebase.perf.metrics.Trace;
+
+public class StartupTimer extends ReactContextBaseJavaModule {
```

Comment:
NAB: Actually, my only comment is that this could be made generic, so that each timing trace doesn't require a new Java class. But I'm sure you're aware of this, and it's not necessary until we add the second trace.

Reply:
Fair point, I guess my thinking here was that we really only need a single timer in the native layer (when the app starts) for now. If we need to do performance traces with firebase we can do that via the JS later and this package

https://rnfirebase.io/perf/usage#custom-tracing